app/test_main.py

```diff
@@ -1,3 +1,43 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestCatAndDogYears:
```

Comment:
why did you decide to use a class?

Reply:
because I want to get used  to the best practice